src/main/java/com/datadoghq/sketch/ddsketch/mapping/LogLikeIndexMapping.java

```diff
@@ -48,15 +39,36 @@
    *     bound of the bucket of index 0
    */
   LogLikeIndexMapping(double gamma, double indexOffset) {
+    this.gamma = requireValidGamma(gamma);
+    this.indexOffset = indexOffset;
+    this.multiplier = Math.log(base()) / Math.log1p(gamma - 1);
+    this.relativeAccuracy = relativeAccuracy(gamma, correctingFactor());
+  }
+
+  private static double relativeAccuracy(double gamma, double correctingFactor) {
+    double correctedGamma = Math.pow(gamma, correctingFactor);
+    return (correctedGamma - 1) / (correctedGamma + 1);
+  }
+
+  static double gamma(double relativeAccuracy, double correctingFactor) {
+    return Math.pow((1 + relativeAccuracy) / (1 - relativeAccuracy), 1 / correctingFactor);
```

Comment:
A comment about what this is achieving, and how, might be nice :)

Reply:
I added comments in 783b37c96dcc79b04a3cdee8dc2eb7eaa587a1bb. I think I'll eventually refactor this code (following the work on `sketches-rs`) to remove some unnecessary complexity and make things clearer to understand (e.g., I believe the correcting factor can actually be expressed by a Lipschitz constant, which is more meaningful).